### DISCUSSION TITLE
Successive Halving Classifier-Budget, eta parameters

### DISCUSSION THREAD

**Discussion by medha-chippa** [online-ml/river#591] [Q&A]
Need some guidance in setting the value of budget, eta in a Successive Halving Classifier.
I'm trying to perform hyperparameter tuning on the HoeffdingTreeClassifier using the Successive Halving Classifier. Im looking for some guidance in this regard. Im looking for an example code snippet that uses the Successive Halving Classifier for tuning the parameters of a classifier along with some explanation on the same. Looking for some guidance in setting the budget parameter when the no of observations/samples in the dataset are finite.

Thank you

**Comment by MaxHalford** [comment]
Hello, I apologize for answering with such delay.

Is this still an issue? Have you made learnings and progress by yourself?
